_posts/2019-03-22-cockpit-190.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cockpit 190
+author: kkoukiou
+date: 2019-03-22
+tags: cockpit linux
+slug: cockpit-190
+category: release
+summary: Cockpit with new features in logs and machines
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 190.
+
+### Logs: Filter log entries by service
+
+![Filtering logs by service](/images/logs-filter-service-name.png)
+
+Logs page allows now to filter the logs results by a specific service.
+
+### Machines: Support for Pausing/Resuming VMs
+
+![Pause and Resume operations for VMs](/images/machines-pause-resume.png)
+
+You can now pause a running VM or resume a paused a VM.
+
+_Thanks to Simon Kobyda for this feature!_
+
+### Machines: Make Autostart property of a Virtual Network configurable
+
+![Autostart property for Virtual Networks](/images/machines-virtual-network-autostart.png)
+
+_Thanks to Simon Kobyda for this feature!_
+
+### Machines: Support for creating VM with option to boot from PXE
+
+You can now choose Network boot when creating a new VM. Supported sources are
+libvirt Virtual Networks and host network devices used with direct assignment.
```

Comment:
I don't really understand what "used with direct assignment" means.

Reply:
https://libvirt.org/formatdomain.html#elementsNICSDirect
```
Provides direct attachment of the virtual machine's NIC to the given physical interface of the host. Since 0.7.7 (QEMU and KVM only)
This setup requires the Linux macvtap driver to be available.
```

Reply:
@martinpitt do you think that's technicality?

Reply:
I see, so the VM will directly "own" the hardware device. Makes sense, thanks! Feel free to actually turn that into a link.

Reply:
Added link